requirements.txt

```diff
@@ -0,0 +1,5 @@
+six
+Django>=1.10
+opentracing<2.0
+django_opentracing>=1.0
+wavefront_pyformance
```

Comment:
Please add versions for `six` and `wavefront-pyformance` and sort the list alphabetically